app/domain/logs.rb

```diff
@@ -591,106 +591,106 @@ module AuthnJwt
       )
 
       ClaimsDenyListValue = ::Util::TrackableLogMessageClass.new(
-        msg: "Claims denylist value is '{0-deny-claims-list}'",
+        msg:  "Claims denylist value is '{0-deny-claims-list}'",
         code: "CONJ00117D"
       )
 
       ValidatingClaimName = ::Util::TrackableLogMessageClass.new(
-        msg: "Validating claim name '{0-claim-name}'",
+        msg:  "Validating claim name '{0-claim-name}'",
         code: "CONJ00118D"
       )
 
       ValidatedClaimName = ::Util::TrackableLogMessageClass.new(
-        msg: "Successfully validated claim name '{0-claim-name}'",
+        msg:  "Successfully validated claim name '{0-claim-name}'",
         code: "CONJ00119D"
       )
       ParsingMandatoryClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Parsing mandatory claims value '{0-mandatory-claims}'",
+        msg:  "Parsing mandatory claims value '{0-mandatory-claims}'",
```

Comment:
why do we need "value'
Can we just say "Parsing mandatory claims..." ??

Reply:
I prefer to keep the value in the message.
I've removed the "value" word